frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -109,9 +93,17 @@ export const BuySell = ({
   };
 
   const buildInfo = (exchange: exchangesAPI.ExchangeDeals): TInfoContentProps => {
-    const cardFee = exchange.deals && exchange.deals.find(feeDetail => feeDetail.payment === 'card')?.fee;
-    const bankTransferFee = exchange.deals && exchange.deals.find(feeDetail => feeDetail.payment === 'bank-transfer')?.fee;
-    return { info: exchange.exchangeName, cardFee, bankTransferFee };
+    const cardFee = exchange.deals.find(feeDetail => feeDetail.payment === 'card')?.fee;
+    const bankTransferFee = exchange.deals.find(feeDetail => feeDetail.payment === 'bank-transfer')?.fee;
+    const bancontactFee = exchange.deals.find(feeDetail => feeDetail.payment === 'bancontact')?.fee;
+    const sofortFee = exchange.deals.find(feeDetail => feeDetail.payment === 'sofort')?.fee;
```

Comment:
Another one, sorry: I think that this could scale better by refactoring the payment methods fees as a map.

Reply:
Not a blocker, but I think it could simplify the code a bit

Reply:
yeah this function just got a bit more ugly :)